### PR TITLE
Fix the problem causing the navbar to be hidden in bigger screen sizes

### PR DESCRIPTION
**Pull request by mpourismaiel**
**What this PR does / why we need it**:
Regression - Previous PR caused the navbar to be hidden in bigger screen sizes

### CODE REVIEW

File: layouts/partials/fragments/nav.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
     </button>
-    <div class="navbar-collapse justify-content-end" id="navbarCollapse">
+    <div class="navbar-collapse justify-content-end collapse show" id="navbarCollapse">
```
Comment (stp-ip) [inline layouts/partials/fragments/nav.html]: `<div class="navbar-collapse justify-content-end d-sm-none" id="navbarCollapse">`

File: static-main/styles/syna.scss [hunk 2]
```diff
@@ -96,14 +96,16 @@
...
-  max-height: 0px;
-  transition: max-height 0.5s cubic-bezier(0, 1, 0, 1);
-  overflow: hidden;
-  display: block;
-}
+@include media-breakpoint-down(md) {
+  .navbar-collapse {
+    max-height: 0px;
+    transition: max-height 0.5s cubic-bezier(0, 1, 0, 1);
+    overflow: hidden;
```
Comment (stp-ip) [inline static-main/styles/syna.scss]: Remove `overflow: hidden;`

**Comment by mpourismaiel** [comment]
@stp-ip Did as you said and had to add `imortant` to the `display: block;` in `.navbar-collapse`. Is that okay?

**Comment by stp-ip** [comment]
Your fix was better. Mine is broken. Thanks for testing.

**Pull request merged**